eng/Subsets.props

```diff
@@ -640,8 +640,8 @@
             <_BuildCoreCLRRuntimePack Condition="'$(RuntimeFlavor)' == 'CoreCLR' and '$(CoreCLRSupported)' == 'true'">true</_BuildCoreCLRRuntimePack>
             <_BuildMonoRuntimePack Condition="'$(RuntimeFlavor)' == 'Mono' and '$(MonoSupported)' == 'true'">true</_BuildMonoRuntimePack>
             <_BuildHostPack Condition="'$(RuntimeFlavor)' == '$(PrimaryRuntimeFlavor)' and '$(TargetsMobile)' != 'true'">true</_BuildHostPack>
-            <_BuildCdacPack Condition="'$(_CDacToolsBuilt)' == 'true' and '$(RuntimeFlavor)' == 'CoreCLR' and '$(TargetsMobile)' != 'true' and '$(TargetsLinuxMusl)' != 'true' and ('$(TargetOS)' == 'windows' or '$(TargetOS)' == 'osx' or '$(TargetOS)' == 'linux')">true</_BuildCdacPack>
-            <_BuildCdacPack Condition="'$(DotNetBuildSourceOnly)' == 'true' or '$(TargetArchitecture)' == 'arm' or '$(TargetArchitecture)' == 'armel' or '$(TargetArchitecture)' == 'x86' or '$(TargetArchitecture)' == 'riscv64'">false</_BuildCdacPack>
+            <_BuildCdacPack Condition="'$(_CDacToolsBuilt)' == 'true' and '$(RuntimeFlavor)' == 'CoreCLR' and '$(TargetsMobile)' != 'true' and ('$(TargetOS)' == 'windows' or '$(TargetOS)' == 'osx' or '$(TargetOS)' == 'linux')">true</_BuildCdacPack>
+            <_BuildCdacPack Condition="'$(DotNetBuildSourceOnly)' == 'true' or '$(TargetArchitecture)' == 'armel' or '$(TargetArchitecture)' == 'riscv64'">false</_BuildCdacPack>
```

Comment:
```suggestion
```

We don't need this one-off condition. There are six other community platforms (freebsd-x64/arm64, linux-loongarch64, illumos-x64, solaris-x64 and haiku-x64) which are on the same plan as linux-riscv64/x86/arm.

Reply:
You mean that we should create a package, but they are not under "official" support?

Reply:
I mean just delete this line and let it evolve freely. Our "normal" way (under src/installers/pkg) is that we always build for "current RID" regardless of it being official or not.

Full source-build:
riscv: https://github.com/am11/dotnet-riscv/actions/runs/16078187257 and loongarch: https://github.com/am11/dotnet-riscv/actions/runs/16078200133

although there is no loongarch64 in explicit-false condition.

Also, linux-x86 is not MSFT official platform RID so it is the same category.

Reply:
The intent was to light it up for whatever platform we were ready to test on. @hoyosjs, any preference?

Reply:
If it's not blocking and it's not too expensive, I don't see a reason not to do light up for community as long as it's understood as community support.

Reply:
Ok, done!